packages/pn-personafisica-webapp/src/component/Contacts/SpecialContacts.tsx

```diff
@@ -154,45 +171,66 @@ const SpecialContacts = ({ recipientId, legalAddresses, courtesyAddresses }: Pro
     initialValues,
     validateOnMount: true,
     validationSchema,
-    onSubmit: (values) => {
-      initValidation(
-        values.addressType,
-        values.s_pec || values.s_mail || internationalPhonePrefix + values.s_phone,
-        recipientId,
-        values.sender,
-        async (status: 'validated' | 'cancelled') => {
-          if (status === 'validated') {
-            // reset form
-            formik.resetForm();
-            await formik.validateForm();
+    onSubmit: (values: SpecialContacts) => {
```

Comment:
no needs to specify type SpecialContacts

Reply:
do it